feature_server/config/scripts/afk.py

```diff
@@ -8,7 +8,7 @@
 from operator import attrgetter
 from twisted.internet import reactor
 from pyspades.common import prettify_timespan
-from commands import name, get_player, add, admin
+from piqueserver.commands import name, get_player, add, admin
```

Comment:
I do agree that it is The Right Way To Move Forward.

but are we keeping the backwards compatibility? I think it is important for people to be able just plug in the scripts from their existing pyspades/pysnip installation without tweaking imports